plugins/localfile/localfile_test.go

```diff
@@ -0,0 +1,94 @@
+package localfile
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stripe/veneur/samplers"
+)
+
+type badWriter struct{}
+
+func (w *badWriter) Write(b []byte) (n int, err error) {
+	return 0, fmt.Errorf("This writer fails when you try to write!")
+}
+
+func TestName(t *testing.T) {
+	plugin := Plugin{FilePath: "doesntexist.txt", Logger: logrus.New()}
+	assert.Equal(t, plugin.Name(), "localfile")
+}
+
+func TestAppendToWriter(t *testing.T) {
+	b := &bytes.Buffer{}
+
+	metrics := []samplers.DDMetric{
+		samplers.DDMetric{
+			Name: "a.b.c.max",
+			Value: [1][2]float64{[2]float64{1476119058,
```

Comment:
Can we get each entry onto its own line? Ditty for the `Tags` field.

(I realize it also `gofmt`s this way, but between the two, I prefer the latter.)